### PR TITLE
Unify config

### PR DESCRIPTION
**Pull request by sparkoo**
This patch removes mandatory registry key on Windows. It now does not fail, but rather uses font in repo and overviews folder as default parameters. IMHO with this it would be easier to dev and easier do distribute without installator on windows. It also now fails faster when font or overviews directory does not exist.

I've also added script do help download overviews to local folder.

Linux and Mac main files are equivalent, I've just removed some duplication. I've tested on Mac.

### CODE REVIEW

File: overviews.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+curl -o overviews/de_mirage.jpg "https://i.imgur.com/6HLCCp1.jpeg"
+curl -o overviews/de_vertigo.jpg "https://i.imgur.com/q8HmCHN.jpeg"
+curl -o overviews/de_vertigo_lower.jpg "https://i.imgur.com/qCbHmJ2.jpeg"
+curl -o overviews/de_nuke.jpg "https://i.imgur.com/frbUQtK.jpeg"
+curl -o overviews/de_nuke_lower.jpg "https://i.imgur.com/vaF05M5.jpeg"
+curl -o overviews/de_cache.jpg "https://i.imgur.com/KqtNKUc.jpeg"
+curl -o overviews/de_inferno.jpg "https://i.imgur.com/MW7Kp7f.jpeg"
+curl -o overviews/de_train.jpg "https://i.imgur.com/FSLMR6V.jpeg"
+curl -o overviews/de_dust2.jpg "https://i.imgur.com/OrsYdDq.jpeg"
+curl -o overviews/de_ancient.jpg "https://i.imgur.com/2ZWUOt3.jpeg"
```
Comment (augustoccesar) [inline overviews.sh]: Maybe could get the overviews from https://github.com/zoidbergwill/csgo-overviews ? It seems to be a nice repo to rely on for this and probably easier to get updated maps.
What do you think about it?

Comment (sparkoo) [reply overviews.sh]: yes, I've updated the script

**Comment by Linus4** [comment]
Hi @sparkoo ! This is a great improvement, thanks a lot for your PR :)

Regarding the csgo-overviews repo: this is where the installer downloaded the images from before I switched to the imgur links. ([NSI Script](https://github.com/Linus4/csgoverview/blob/cffab2027bd358bba2a2da221e1fc034d3767923/windows_installer.nsi#L62)) As you can see, de_ancient was added just 3 months ago. I guess it would be better to use the images from Github because they have a URL that stays the same after an update, so it does not have to be changed in the install scripts. But using our own images means that we are independent from another project. 
Imo we could switch back to using this repo's images as they have been updated. What do you guys think? @augustoccesar @sparkoo

**Comment by sparkoo** [comment]
The `zoidbergwill/csgo-overviews` repo looks quite stable, so I wouldn't be worried it will dissapear and I don't see new maps coming either in following years so I think it's safe to use it.
btw installer bundle overviews or does it download them on install ?

**Comment by Linus4** [comment]
Ok in that case, we'll switch back to the overviews from the repo.
The installer downloads the overviews on install. So the links will have to be updated in the nsi script as well.

**Comment by augustoccesar** [comment]
@Linus4 I think is fine to point to the repo as well. But I also can see the point about it taking maybe too long to add new maps or something like that. But I think we can also contribute to that repo in this case by opening PR with a new map when/if it happens.
Not sure what is the speed for PR's to be merged there, but I believe is an overall good idea to depend on a stable repo dedicated to these images.

**Comment by Linus4** [comment]
Thanks again @sparkoo ! I will soon release a new version with your changes in it.

**Pull request merged**